module4/assignment4.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Assignment 4</title>
+    <meta charset="utf-8">
+    <script src="http://code.jquery.com/jquery-1.6.4.min.js"></script>
+    <link rel="stylesheet" href="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+    <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    <link rel="stylesheet" href="_css/theme.css">
+    <link rel="stylesheet" href="_css/brehm_a4.css">
+</head>
+
+<body>
+    <!--    
+        Carter Brehm
+        assignment4.html
+        INFO2439.PLA
+        Thoendel
+        9/7/13
+    -->
+    <div data-role="homePage" data-title="Brehm | INFO 2439 | Assignment #4">
+        <!--Begin Page-->
+        <header data-role="header" data-position="fixed">
+            <!--Begin Header-->
+            <a href="../index.html" data-ajax="false" data-rel="external" data-icon="home">Home</a>
+            <h1>Assignment 4</h1>
+        </header>
+        <!--End Header-->
+        <div data-role="content">
+            <!--Begin Content-->
+            <form action="http://lolberding.portfolios.mccinfo.net/info_survey/bistrosurvey_insert.php" method="post">
+                <label for="meal">How would you rate your meal today?</label>
+                <select name="meal_rate" id="meal_rate">
+                        <option value="1">1 - Poor</option>
+                        <option value="2">2 - Needs Work</option>
+                        <option value="3">3 - Fair</option>
+                        <option value="4">4 - Good</option>
+                        <option value="5">5 - Excellent</option>
+                    </select>
+                <hr>
+                <label for="service">How would you rate your service today?</label>
+                <select name="service_rate" id="service_rate">
+                        <option value="1">1 - Poor</option>
+                        <option value="2">2 - Needs Work</option>
+                        <option value="3">3 - Fair</option>
+                        <option value="4">4 - Good</option>
+                        <option value="5">5 - Excellent</option>
+                    </select>
+                <hr>
+                <label for="experience">How would you rate your experience today?</label>
+                <select name="overall_exp" id="overall_exp">
+                        <option value="1">1 - Poor</option>
+                        <option value="2">2 - Needs Work</option>
+                        <option value="3">3 - Fair</option>
+                        <option value="4">4 - Good</option>
+                        <option value="5">5 - Excellent</option>
+                    </select>
+                <hr>
```

Comment:
The book shows the use of <div> tags to group each label + input pair to focus enhancements and styling.